ansible/roles/node/tasks/localBuildInstall.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Copy node binaries
   copy:
-    src: "{{ localBuildOutput }}/{{ item }}"
+    src: "{{ localBuildOutput }}/{{ item }}" #docker builds, use: kubernetes/_output/dockerized/bin/linux/amd64/{{ item }}
```

Comment:
agree its kinda hacky for OSx  simply cause all the paths are different and dir structure doesnt match.  not sure what the right solution is honestly to make it cross platform.  I wonder wether upstream can just  have one path for build output... so this isnt an issue